packages/devtools/src/constants.js

```diff
@@ -0,0 +1,86 @@
+export const DEFAULT_WIDTH = 1200
+export const DEFAULT_HEIGHT = 900
+export const DEFAULT_X_POSITION = 0
+export const DEFAULT_Y_POSITION = 0
+
+export const ELEMENT_KEY = 'element-6066-11e4-a52e-4f735466cecf'
+
+export const DEFAULT_FLAGS = [
```

Comment:
what do you think on adding `--disable-features=NetworkService` to flags?

Reply:
What does it do?